lib/charms/prometheus_k8s/v0/prometheus_remote_write.py

```diff
@@ -923,9 +923,15 @@ def alerts(self) -> dict:
                 continue
 
             try:
+                # remote write doesn't have scrape metadata; need to get topology from somewhere
+                # scrape_metadata = json.loads(relation.data[relation.app]["scrape_metadata"])
+                # topology = JujuTopology.from_relation_data(scrape_metadata)
+                topology = JujuTopology(
+                    "dont", "really", "know", "yet"
```

Comment:
Just a note to self that we need to change this to something proper before merging.

Reply:
You fixed it :) resolving